src/Components/About.js

```diff
@@ -0,0 +1,307 @@
+import React, { useEffect } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+import aos from "aos";
+import "aos/dist/aos.css";
+import { useMediaQuery } from "react-responsive";
+
+//import pictures nightmare
+import whatsapp from "../assets/logos/whatsapp.svg";
+import calender from "../assets/logos//calendar.svg";
+import email from "../assets/logos//envelope.svg";
+import telegram from "../assets/logos//telegram.svg";
+import location from "../assets/logos//location.svg";
+import linkedIn from "../assets/logos//linkedin.svg";
+import github from "../assets/logos//github.svg";
+import facebook from "../assets/logos//facebook.svg";
+import leetcode from "../assets/logos//leetcode.svg";
+
+function About() {
+  const isSmallScreen = useMediaQuery({ query: "(max-width: 359px)" });
+  let output;
+  useEffect(() => {
+    aos.init({ duration: 2000 });
+  }, []);
+  if (isSmallScreen) {
+    output = <SmallScreen />;
+  } else {
+    output = <Normal />;
+  }
+  return output;
+}
+function SmallScreen() {
+  return (
+    <Container
+      data-aos="fade-up"
+      id="about"
+      style={{
+        color: "#20C20E",
+        fontFamily: "Share Tech Mono",
+      }}
+    >
+      <Row>
+        <Container>
+          <Col>
+            <h2>About me</h2>
+            <p>
+              Hello! I'm Chai Ee. I enjoy tinkering around with
+              electronics/software and finding out how things work.
+            </p>
+            <p>
+              I like coding, especially backend, and am passionate about secured
+              websites and optimizing algorithms. I always seek to look for ways
+              that we can improve our products. I also like tinnkering around
+              electronic, currently ddoing an Arduino Project
+            </p>
+            <p>
+              Currently, I am an undergraduate at Nanyang Technological
+              University, studying Electrical and Electronic Engineering. I'm
+              looking for exciting opportunities to gain professional exposure
+              in the Software/IT Industry to jumpstart my career!
+            </p>
+          </Col>
+        </Container>
+        <Container>
+          <Col id="Bio">
+            <Row>
+              <h2>Bio</h2>
+              <Col>
+                <div>
+                  <img src={calender} alt="age" />
+                  <span style={{ marginLeft: "6px" }}>age: 22</span>
```

Comment:
inline styles are pretty much a no-go :)! You should either use your bootstrap classes, or custom css.